cms/djangoapps/contentstore/tests/test_course_settings.py

```diff
@@ -179,6 +180,29 @@ def compare_date_fields(self, details, encoded, context, field):
         elif field in encoded and encoded[field] is not None:
             self.fail(field + " included in encoding but missing from details at " + context)
 
+    @ddt.data(
+        (False, False),
+        (True, True),
+        (True, True),
```

Comment:
Duplicate test? Probably meant to be `(True, False)`

Reply:
Hah, thanks. Duh.